src/main/catalog-sources/kubeconfig-sync.ts

```diff
@@ -24,7 +24,7 @@ export class KubeconfigSyncManager extends Singleton {
   protected static readonly syncName = "lens:kube-sync";
 
   @action
-  startSync(port: number): void {
+  startSync(): void {
```

Comment:
Passing port multiple layers down to just for `cluster.init()` felt wrong. Init is anyway called from `ClusterManager`.

Reply:
So I have noticed that all of `Cluster.init()` can be hoisted into `Cluster.constructor`.

Reply:
Also I specifically didn't want to rely on `ClusterManager`. Wouldn't it be possible to query this for a cluster ID instead?